lib/Compiler.js

```diff
@@ -340,6 +341,7 @@ class Compiler extends Tapable {
 						source.existsAt = targetPath;
 						source.emitted = true;
 						this.outputFileSystem.writeFile(targetPath, content, callback);
+						source._cachedSource = undefined;
```

Comment:
Assigning `_cachedSource` to any type of `source` may change its hidden class if it's not a `CachedSource`, so there has to be at least a check if the property was there before clearing it. Also, is the initial value to which this code resets `undefined`, or is it `null`?

Reply:
That's a good point :+1:

The initial value is `undefined`: https://github.com/webpack/webpack-sources/blob/master/lib/CachedSource.js#L13